applications/utils/export_data.py

```diff
@@ -0,0 +1,167 @@
+from csv import writer
+from pathlib import Path
+from typing import List
+
+from django.conf import settings
+from django.utils import timezone
+
+from ..models import ApplicationEvent, ApplicationEventSchedule, EventReservationUnit
+
+
+class ApplicationDataExporter:
+    @classmethod
+    def export_application_data(cls) -> None:
+        now = timezone.now()
+        root = Path(settings.BASE_DIR)
+        file_name = f"application_data_{now.strftime('%d-%m-%Y')}.csv"
+        path = root / "exports"
+        path.mkdir(parents=True, exist_ok=True)
+
+        application_events: List[ApplicationEvent] = list(
+            ApplicationEvent.objects.all()
```

Comment:
I think we need to get application events per application round. So we'll need to parametrize this `ApplicationDataExporter.export_application_data` OR set this be stateful and create an instance of this and set the application_round there and then call this `export_application_data` (and change it to instance method). 
I'm perhaps more towards the parameterized classmethod call  but you can decide.

Reply:
I elected for parameterization as I believe this is the better option as well :smiley: